tests/regression/test_poisson_mixed_strong_bcs.py

```diff
@@ -27,13 +27,29 @@
 from firedrake import *
 
 
-def poisson_mixed(size, parameters={}):
+def poisson_mixed(size, parameters={}, quadrilateral=False):
     # Create mesh
-    mesh = UnitSquareMesh(2 ** size, 2 ** size)
+    mesh = UnitSquareMesh(2 ** size, 2 ** size, quadrilateral=quadrilateral)
 
     # Define function spaces and mixed (product) space
-    BDM = FunctionSpace(mesh, "BDM", 1)
-    DG = FunctionSpace(mesh, "DG", 0)
+    if quadrilateral:
+        S0 = FiniteElement("CG", "interval", 1)
+        S1 = FiniteElement("DG", "interval", 0)
+
+        T0 = FiniteElement("CG", "interval", 1)
+        T1 = FiniteElement("DG", "interval", 0)
+
+        DG_elt = OuterProductElement(S1, T1)
+        BDM_elt_h = HDiv(OuterProductElement(S1, T0))
```

Comment:
You could make a "BDM-like" element by taking HDiv(CG1xDG1) + HDiv(DG1xCG1), though this isn't "the" BDM element on quads.  If this works, do it.  If it doesn't work, stick with the RT element and change some variable names.